persistence/sql/migrations/sql/20220819132309000000_add_flow_type_to_identity_recovery_tokens.cockroach.up.sql

```diff
@@ -0,0 +1,17 @@
+ALTER TABLE identity_recovery_tokens
```

Comment:
Please remove the COMMIT; BEGIN transaction statements and move the DDL and DSL statements into separate files

Reply:
Done.